src/main/scala/it/pagopa/interop/catalogprocess/api/impl/ProcessApiServiceImpl.scala

```diff
@@ -224,46 +221,28 @@ final case class ProcessApiServiceImpl(
       createEServiceDocumentResponse[EService](operationLabel)(createEServiceDocument200)
     }
   }
+
   override def getEServiceDocumentById(eServiceId: String, descriptorId: String, documentId: String)(implicit
     contexts: Seq[(String, String)],
-    toEntityMarshallerProblem: ToEntityMarshaller[Problem],
-    toEntityMarshallerFile: ToEntityMarshaller[File]
+    toEntityMarshallerEServiceDoc: ToEntityMarshaller[EServiceDoc],
+    toEntityMarshallerProblem: ToEntityMarshaller[Problem]
   ): Route = authorize(ADMIN_ROLE, API_ROLE, SECURITY_ROLE, M2M_ROLE) {
     val operationLabel =
       s"Retrieving EService document $documentId for EService $eServiceId and descriptor $descriptorId"
     logger.info(operationLabel)
 
-    val result: Future[DocumentDetails] = for {
-      document    <- catalogManagementService.getEServiceDocument(eServiceId, descriptorId, documentId)
-      contentType <- getDocumentContentType(document)
-      response    <- fileManager.get(ApplicationConfiguration.storageContainer)(document.path)
-    } yield DocumentDetails(document.name, contentType, response)
+    val result: Future[EServiceDoc] = catalogManagementService
+      .getEServiceDocument(eServiceId, descriptorId, documentId)
+      .map(Converter.convertToApiEserviceDoc)
 
     onComplete(result) {
-      getEServiceDocumentByIdResponse[DocumentDetails](operationLabel) { documentDetails =>
-        val output: MessageEntity = convertToMessageEntity(documentDetails)
-        complete(output)
+      getEServiceByIdResponse[EServiceDoc](operationLabel) { documentMetadata =>
+        logger.info(s"E-Service document metadata with id ${documentMetadata.id}")
```

Comment:
the success log is generally not required, because the successful response is automatically logged for each endpoint.
An exception could be when it generates an id (e.g. a creation endpoint) and we want to trace it with logs